src/main/java/yasp/Parse.java

```diff
@@ -291,15 +295,11 @@ public void onCombatLogEntry(Context ctx, CombatLogEntry cle) {
     }
 
     @OnEntityEntered
-    public void onEntityEntered(Context ctx, Entity e) {
-        processEntity(ctx, e, false);
-    }
-    
     @OnEntityLeft
-    public void onEntityLeft(Context ctx, Entity e) {
-        processEntity(ctx, e, true);
+    public void onEntityExistenceChanged(Context ctx, Entity e) {
```

Comment:
Since the annotation here is still OnEntityLeft, I think the method name should reflect that.